openunmix/model.py

```diff
@@ -39,6 +44,22 @@ def __init__(
         self.n_hop = n_hop
         self.center = center
 
+        # alt with asteroid
+        self.use_asteroid = use_asteroid
+        if use_asteroid :
+            dft_filters = torch_stft_fb.TorchSTFTFB(
+                        n_filters=n_fft,
+                        kernel_size=n_fft,
+                        stride=n_hop,
+                        window=torch.hann_window(n_fft).numpy(),
```

Comment:
that would need to be a torch window. Why not use `torch.window`?

Reply:
We can use a window with a tensor type, but asteroid converts the window into numpy anyway.

Reply:
well, but then leave this to asteroid... your current code doesn't work. Check the jit test output

`
E           Tried to access nonexistent attribute or method 'numpy' of type 'Optional[Tensor]'.:
```